app/User.php

```diff
@@ -15,7 +15,9 @@ final class User extends Authenticatable
     use HasTimestamps, ModelHelpers, Notifiable;
 
     const DEFAULT = 1;
+
     const MODERATOR = 2;
+
```

Comment:
Maybe revert the whitelines between constants that were added. Not sure why shift adds these.